src/main/java/net/dries007/tfc/compat/jade/common/BlockEntityTooltips.java

```diff
@@ -378,7 +378,7 @@ else if (pot.getOutput() != null && !pot.getOutput().isEmpty())
             final LoomRecipe recipe = loom.getRecipe();
             if (recipe != null)
             {
-                tooltip.accept(Helpers.translatable("tfc.jade.loom_progress", loom.getProgress(), recipe.getStepCount(), recipe.getResultItem()));
+                tooltip.accept(Helpers.translatable("tfc.jade.loom_progress", loom.getProgress(), recipe.getStepCount(), recipe.getResultItem().getDisplayName().getString()));
```

Comment:
```suggestion
                tooltip.accept(Helpers.translatable("tfc.jade.loom_progress", loom.getProgress(), recipe.getStepCount(), recipe.getResultItem().getDisplayName()));
```
`getString()` shouldn't be required, nor should it exist, as translation components get replaced inline.